os/fs/driver/mtd/smart.c

```diff
@@ -5476,9 +5476,9 @@ static int smart_journal_recovery(FAR struct smart_struct_s *dev, journal_log_t
 			fvdbg("Entire of sector is valid!!!!\n");
 			ret = smart_journal_checkout(dev, log, address);
 			if (ret != OK) {
-				/* If checkout failed, keep last journal */
-				return ret;
+				fdbg("Sector was valid but checkout failed\n");
```

Comment:
@Taejun-Kwon how about this?
```
if (ret == OK) {
			fvdbg("Entire of sector is valid!!!!\n");
			ret = smart_journal_checkout(dev, log, address);
			if (ret != OK) {
				fdbg("Sector was valid but checkout failed\n");
			}
			return ret;
		}
		
		fvdbg("Entire of sector is invalid\n");
```

Reply:
yeah more good. I will modify it. thanks.